tiatoolbox/models/architecture/nuclick.py

```diff
@@ -353,7 +357,8 @@ class NuClick(ModelABC):
 
     """
 
-    def __init__(self, num_input_channels: int, num_output_channels: int):
+    def __init__(self, num_input_channels: int, num_output_channels: int) -> None:
+        """Initializes :class:`NuClick`."""
```

Comment:
```suggestion
        """Initialize :class:`NuClick`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090